pkg/scm/git/git_test.go

```diff
@@ -115,6 +116,127 @@ func TestValidCloneSpecRemoteOnly(t *testing.T) {
 	}
 }
 
+func TestMungeNoProtocolURL(t *testing.T) {
```

Comment:
yep - testing of the new URLMods field - good catch.  As far as where to test, the TestMungeNoProtocolURL already confirms that with the no protocol file URL, the path is not prepended with "file://"

However, I think I should add a test for ParseFile in the git dir, and tweak the TestDownloaderForSource test to ensure the new field (and its resulting mods) are correct.